command/src/main/java/com/jagrosh/jdautilities/command/SlashCommand.java

```diff
@@ -177,6 +178,24 @@ public void onAutoComplete(CommandAutoCompleteInteractionEvent event) {}
      */
     @Override
     protected void execute(CommandEvent event) {}
+    
+    /**
+     * Returns the description of this SlashCommand instance.<br>
+     * Should there be {@link #getDescriptionLocalization() translations for this text} will the client try to
+     * obtain a translation for the Description using {@link TranslateUtil#getDefaultLocale() the default locale} and
+     * in case of a null or empty String return the default help text set.
+     * 
+     * @return Translated Help text for default locale, or help text set in constructor.
+     */
+    @Override
+    public String getHelp(){
+        String helpMessage = null;
+        if (!getDescriptionLocalization().isEmpty()) {
+            helpMessage = getDescriptionLocalization().get(TranslateUtil.getDefaultLocale());
+        }
+        
+        return (helpMessage == null || helpMessage.isEmpty()) ? this.help : helpMessage;
```

Comment:
We should always return this.help if it is specified, and only do this logic if it's not

Reply:
Sorry for a late reply... Honestly forgot this.

The main issue is, that help is always there. By default is it `no help available`, so if we should have this logic will this require modifications to the default help message (i.e. make it null and have `getHelp()` return the default text in case of null), which I feel like a breaking change here.

Maybe another aproach could be a dedicated option and getter for the command description for slash commands that defaults to returning `getHelp()` if nothing is set?

Reply:
Any input on this?